qiskit/transpiler/passes/optimization/collect_multiqubit_blocks.py

```diff
@@ -23,7 +23,7 @@ class CollectMultiQBlocks(AnalysisPass):
     by any single group of gates
 
     Traverse the DAG and find blocks of gates that act consecutively on
-    groups of qubits. Write the blocks to propert_set as a list of blocks
+    groups of qubits. Write the blocks to property_set as a list of blocks
```

Comment:
I see the typo here, but here and in the rest of this docstring it feels weird to say "property_set" in prose (and "max_block_size", etc) with the underscores but without the code formatting.  Do you prefer to make these "property set" (regular English) or "` ``property_set`` `" with the code format?

Reply:
all of the in code format, inf654a78864ab779c6e10a5e5f4f5faeb8e941768